src/examples/grid/GridVariable.tsx

```diff
@@ -6,34 +6,40 @@
  */
 
 import React from 'react';
+import styled from 'styled-components';
+import { getColor } from '@zendeskgarden/react-theming';
 import { MD } from '@zendeskgarden/react-typography';
-import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Grid } from '@zendeskgarden/react-grid';
+
+const StyledGrid = styled(Grid)`
+  color: ${p => getColor({ variable: 'foreground.default', theme: p.theme })};
```

Comment:
The approach I used is to [set](https://github.com/zendeskgarden/website/pull/626/files#diff-02a2ac8dc037c29bb3d8cb4e69ea2535fee5c833fbd580f188015f100f0b33f9R131) the text color at the `CodeSandbox` level. This aligns with `react-components`' Storybook [preview.js](https://github.com/zendeskgarden/react-components/blob/next/.storybook/preview.js#L52). 

@jzempel Any preference?